docs/quick_start.md

```diff
@@ -0,0 +1,103 @@
+# Quick Start
```

Comment:
’Quick Start‘ or 'Quickstart' ?

Reply:
use `Quickstart` to keep consistent with other docs.